statistics/histogram.go

```diff
@@ -374,6 +374,34 @@ func (hg *Histogram) RemoveIdxVals(idxValCntPairs []TopNMeta) {
 	}
 }
 
+// AddIdxVals adds the given values from the histogram.
+func (hg *Histogram) AddIdxVals(idxValCntPairs []TopNMeta) {
+	totalAddCnt := int64(0)
+	for bktIdx, pairIdx := 0, 0; bktIdx < hg.Len(); bktIdx++ {
+		for pairIdx < len(idxValCntPairs) {
+			// If the current val smaller than current bucket's lower bound, skip it.
+			cmpResult := bytes.Compare(hg.Bounds.Column(0).GetBytes(bktIdx*2), idxValCntPairs[pairIdx].Encoded)
+			if cmpResult > 0 {
+				continue
+			}
+			// If the current val bigger than current bucket's upper bound, break.
+			cmpResult = bytes.Compare(hg.Bounds.Column(0).GetBytes(bktIdx*2+1), idxValCntPairs[pairIdx].Encoded)
+			if cmpResult < 0 {
+				break
```

Comment:
Why break instead of continue? Can we assume that `idxValCntPairs` is ordered here?